### DISCUSSION TITLE
🧩 FEATURE REQUEST: There Must be Another Way to Gain Reputation for Readers

### DISCUSSION THREAD

**Discussion by mekadi** [dailydotdev/daily#1681] [Feature Requests]
### Concise Description

Reputation System is Not Fair

### Description

Earning reputation is currently one-sided, benefiting only writers. This creates an unfair system where readers, who may actively engage with the platform, cannot gain reputation. For instance, I lost my 67-day streak, and I wasn't able to recover it because I don't have enough reputation but meanwhile writers with enough reputation can easily restore their streaks and this feels unfair. Additionally, I cannot change my postcard color because I'm not a writer with enough reputation.

### Solution Proposed

Readers should also have opportunities to earn reputation. For example, users could gain reputation by reaching a specific reading streak or by reaching a certain number of articles read.

### Code of Conduct

- [ ] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @mekadi,

Reputation reflects the contribution of a user to the community. According to the reputation design, the community has to grant you points, hence reading posts isn't enough to get reputation. I understand your frustration though, and we also consider other ways that can help restore your streak
